packages/hooks/__tests__/use-locale.vitest.tsx

```diff
@@ -1,46 +1,43 @@
-import { h, nextTick, computed } from 'vue'
+import { nextTick, computed, defineComponent } from 'vue'
 import { mount } from '@vue/test-utils'
+import { describe, it, expect, beforeEach, afterEach } from 'vitest'
 import Chinese from '@element-plus/locale/lang/zh-cn'
 import English from '@element-plus/locale/lang/en'
 import { useLocale, buildTranslator } from '../use-locale'
 import { provideGlobalConfig } from '..'
+import type { Language } from '@element-plus/locale'
+import type { PropType, ComponentPublicInstance } from 'vue'
+import type { VueWrapper } from '@vue/test-utils'
 
-const TestComp = {
+const TestComp = defineComponent({
   setup() {
     const { t } = useLocale()
-    return () => {
-      return h(
-        'div',
-        { class: 'locale-manifest' },
-        t('el.popconfirm.confirmButtonText')
-      )
-    }
+    return () => (
+      <div class="locale-manifest">{t('el.popconfirm.confirmButtonText')}</div>
+    )
   },
-}
+})
 
 describe('use-locale', () => {
-  let wrapper
+  let wrapper: VueWrapper<ComponentPublicInstance>
+
   beforeEach(() => {
     wrapper = mount(
       {
         props: {
-          locale: Object,
+          locale: Object as PropType<Language>,
         },
         components: {
           'el-test': TestComp,
         },
         setup(props, { slots }) {
           provideGlobalConfig(computed(() => ({ locale: props.locale })))
-          return () => slots.default()
+          return () => slots.default?.()
```

Comment:
**Question:**  Can we use `jsx` in here, just to keep consistency.

Reply:
I don't think it's necessary, it's just one line. If we use JSX, it just adds a bunch of angle brackets to the existing code.
```tsx
<>slots.default?.()</>
```